src/components/checkbox/checkbox.scss

```
@@ -9,9 +9,12 @@
 @include checkbox.core-styles;
 @include form-field.core-styles;
 
+$constructed-size-of-checkbox: 2.5rem; // MDC has some variables that control the size of the
+// checkbox component,but they will be constructed to `40px` (`2.5rem`) in `height` and` width`
+
 .mdc-form-field {
-    display: flex;
-    flex-wrap: wrap;
+    display: grid;
+    grid-template-columns: var($constructed-size-of-checkbox) 1fr;
```

```suggestion
    grid-template-columns: $constructed-size-of-checkbox 1fr;
```

no need for the var anymore :)